### PR TITLE
Updated output dependencies content for style/consistency

### PR DESCRIPTION
**Pull request by tpoitras**
ping @kargakis for Tech Review, please. :)

updated your PR#225 -- trello card https://trello.com/c/QeyUIKd8

### CODE REVIEW

File: dev_guide/output_deps.adoc [hunk 1]
```diff
@@ -5,50 +5,64 @@
...
 toc::[]
 
 == Overview
-`build-chain` is an experimental command-line tool for OpenShift v3. This tool will start at the imagerepo:tag specified and determine what Builds will be triggered (due to ImageChangeTriggers on the BuildConfig) when this imagerepo:tag is updated. Further, it analyzes the output of those Builds to determine if they in turn update another imagerepo:tag. If they do, the tool continues to follow the dependency tree. It then outputs the total graph of what imagerepo:tags will be impacted by an update to the top level imagerepo:tag. The default output is set to JSON but the user can also choose between DOT and AST.
- 
- 
-== Examples
+*build-chain* is an experimental command-line tool for OpenShift v3, which
+allows you to identify which images would be affected by updating an image in a
+specified image repository.
```
Comment (kargakis) [inline dev_guide/output_deps.adoc]: s/repository/stream. Actually do this for all repository mentions.

**Comment by tpoitras** [comment]
@kargakis -- thank you for the review. I've changed:

s/repository/stream
s/repositories/streams

All good now?

**Comment by kargakis** [comment]
Yep. LGTM

**Comment by adellape** [comment]
@tpoitras Though it's been in this state for a while, the topic title is "Dependency tree generation" (non-title case), yet the title in `_build_cfg.yml` (and therefore the title that shows up in the left nav) is "Output Build Dependencies". We should probably pick one and sync them up (using title case).

I'd probably vote for "Dependency Tree Generation" for both. Whatever the choice, we should probably also make sure the filename is sane.

@kargakis Any strong feelings on the title either way?

File: dev_guide/output_deps.adoc [hunk 2]
```diff
@@ -5,50 +5,64 @@
...
+== Usage
 
-Build and output the dependency tree for the specified image repository with 'latest' tag in 'testing' namespace using DOT syntax
-----
-$ openshift ex build-chain [image-repository] -o dot -n testing
-----
+Build a dependency tree for the specified image stream with the specified
+tag in the current namespace and output it in JSON:
+****
+`$ openshift ex build-chain _<image-stream>_:_<tag>_`
```
Comment (adellape) [inline dev_guide/output_deps.adoc]: To render properly, this should be (like the one near the bottom):

`_<image-stream>:<tag>_`

**Comment by kargakis** [comment]
@adellape +1 for "Dependency Tree Generation"

**Comment by tpoitras** [comment]
@adellape & @kargakis -- I totally agree about the naming; thanks for pointing this out, Alex.

Roping in @CowboysFan to sign off on the name change before this gets merged.

**Comment by CowboysFan** [comment]
@adellape @tpoitras @kargakis I agree that we should change the names and sync them up. However, if that topic is about "generating a tree dependency" or "building a tree dependency" then that's what we should call it. It appears to me from the examples that we're showing how to build a tree dependency. So, I'd call it either:

Building Dependency Trees, or
Generating Dependency Trees

@tpoitras Also, the overview at this point seems quite technical, and doesn't really explain what this whole topic is about and what you're trying to accomplish. What is the  imagerepo:tag that's mentioned here? And should that be marked up somehow? I hope we're going to come back and expand that a bit. 

Also, since the examples are only command syntax with one or two sentence descriptions, it sounds like a prime candidate to put those in a table similar to what we did for the CLI topic group. They don't look like real examples at this point.

**Comment by adellape** [comment]
I'd go with "Building Dependency Trees" between those two.

File: _build_cfg.yml [hunk 3]
```diff
@@ -191,8 +191,8 @@ Topics:
     File: application_health
   - Name: Events
     File: events
-  - Name: Output Build Dependencies
-    File: output_deps
+  - Name: Dependency Tree Generation
+    File: dependecy_tree_generation
```
Comment (adellape) [inline _build_cfg.yml]: Typo here and in the actual `dev_guide/dependecy_tree_generation.adoc` file name itself.

File: using_images/sti_images/ruby.adoc [hunk 4]
```diff
@@ -260,6 +260,8 @@ openshift/ruby-20-rhel7
 ====
 
 == Environment Variables
+The Ruby image supports a number of environment variables which can be set to
+control the configuration and behavior of the Ruby runtime.
```
Comment (adellape) [inline using_images/sti_images/ruby.adoc]: Assuming this is a straggler from another PR?

File: _build_cfg.yml (hunk 3 above)
Comment (CowboysFan) [inline _build_cfg.yml]: @adellape I would lean toward Building Dependency Trees based on that intro paragraph:

"Build a dependency tree for the specified image stream......"

However, I'd be okay with either of these over what we have currently.

File: using_images/sti_images/ruby.adoc (hunk 4 above)
Comment (tpoitras) [inline using_images/sti_images/ruby.adoc]: DARN! Yes. I'll fix.

**Comment by tpoitras** [comment]
Should be good now @adellape

**Comment by tpoitras** [comment]
@adellape & @CowboysFan -- I've made some changes to address your comments. If this is good now, please merge.

**Comment by adellape** [comment]
LGTM, thanks. Merging.

**Pull request merged**